todo_rxbloc_app/lib/feature_manage_todo/bloc/manage_todo_bloc.dart

```diff
@@ -0,0 +1,113 @@
+import 'package:rx_bloc/rx_bloc.dart';
+import 'package:rxdart/rxdart.dart';
+import 'package:todos_repository_core/todos_repository_core.dart';
+
+import '../../base/common_blocs/coordinator_bloc.dart';
+import '../../base/enums/current_page_enum.dart';
+import '../../base/enums/new_todo_enum.dart';
+import '../../base/models/navigation_parametars.dart';
+import '../../base/services/manage_todo_service.dart';
+import '../../base/utils/validators/title_validator.dart' as validator;
+import '../../feature_homepage/bloc/navigation_bloc.dart';
+
+part 'manage_todo_bloc.rxb.g.dart';
+part 'manage_todo_bloc_extensions.dart';
+part 'manage_todo_bloc_models.dart';
+
+/// A contract class containing all events of the ManageTodoBloC.
+abstract class ManageTodoBlocEvents {
+  @RxBlocEvent(type: RxBlocEventType.behaviour)
+  void setTitle(String title);
+  @RxBlocEvent(type: RxBlocEventType.behaviour, seed: '')
+  void setDescription(String description);
+  void saveTodo();
+  void updateTodo();
+}
+
+/// A contract class containing all states of the ManageTodoBloC.
+abstract class ManageTodoBlocStates {
+  Stream<String> get validateTitle;
```

Comment:
The props should not contain verbs such as validate, get etc. For more information look at the effective dart documentation.

I would suggest renaming the fields

- validateTitle: title
- getDescription: description
- showErrors: errorVisible
-